schema.sql

```diff
@@ -11,3 +11,48 @@ CREATE TABLE IF NOT EXISTS animals (
 /*Add species column to animals table*/
 ALTER TABLE animals
 ADD COLUMN species varchar(255);
+
+/*Creating table named owners*/
+CREATE TABLE owners (
+  id serial PRIMARY KEY,
+  full_name varchar(255),
+  age integer
+);
+
+/*Creating table named species*/
+CREATE TABLE species (
+  id serial PRIMARY KEY,
+  name varchar(255)
+);
```

Comment:
Your code is correct and meets the project requirements. You have created a table named species with the following columns:

- id: integer (set it as autoincremented PRIMARY KEY) 👍
- name: string 👌

You have used the `serial` data type for the `id` column, which is a PostgreSQL-specific data type that automatically generates a unique integer value for each row. This is equivalent to using the `int` data type with the `IDENTITY(1,1)` property in SQL Server. You have also used the `PRIMARY KEY` constraint to make the `id` column the primary key of the table, which ensures data integrity and efficient data retrieval. 🚀

Your code is well-written and follows the SQL syntax and conventions. You have used appropriate data types and column names for your table. You have also used parentheses and semicolons to enclose and terminate your SQL statements. 🙏

Here is your code as it is:

```sql
CREATE TABLE species (
  id serial PRIMARY KEY,
  name varchar(255)
);
```

I hope this helps you confirm that your code meets the project requirements. If you have any questions, feel free to ask me. 😊👏